### PR TITLE
xrEngine: setting of `rsGL` flag fixed

### PR DESCRIPTION
**Pull request by vTurbine**
Seems the previous fix was incomplete and this one solves the issue. At last.
Also some cosmetics applied.

### CODE REVIEW

File: src/xrEngine/main.cpp [hunk 1]
```diff
@@ -298,6 +297,7 @@ ENGINE_API int RunApplication()
     else
         Console->Execute("renderer renderer_r1");
 
+    Device.Initialize();
```
Comment (Xottab-DUTY) [inline src/xrEngine/main.cpp:301]: It seems that if `CheckBenchmark()` will return true, then Device will be unitialized

Comment (Xottab-DUTY) [reply src/xrEngine/main.cpp:301]: But anyway I just fixed it by moving initialization stages without changing them. Sorry if this was impolite.

**Comment by Xottab-DUTY** [comment]
Since the issue was fixed in b0a2128da0ec71109ddcad798fe28e9b8641f20d you can leave this as code style changing PR if you want.

**Comment by vTurbine** [comment]
This looks better, thanks. I'm closing the PR

**Pull request closed without merging**